templates/pioneer-study/pathfinder-event.1.parquetmr.txt

```diff
@@ -0,0 +1,14 @@
+message event {
+  required group metadata {
+    required int64  Timestamp;
+    required binary documentId (UTF8);
+    required binary pioneerId (UTF8);
+    required binary studyName (UTF8);
+    optional binary geoCity (UTF8);
+    optional binary geoCountry (UTF8);
+    optional binary normalizedChannel (UTF8);
+  }
+  required binary eventId (UTF8);
+  required int64 timestamp;
+  required binary metadata (UTF8);
```

Comment:
Wait, I just realized: you have two fields named metadata in this parquet schema -- this will cause problems when reading the data back in. Please rename this event-specific metadata field